django/contrib/admin/static/admin/js/change_form.js

```diff
@@ -2,8 +2,19 @@
 {
     const inputTags = ['BUTTON', 'INPUT', 'SELECT', 'TEXTAREA'];
     const modelName = document.getElementById('django-admin-form-add-constants').dataset.modelName;
+    const submitButtons = document.querySelectorAll('input[type=submit]');
```

Comment:
Should this be future-proofed by also targetting `button` elements?

Reply:
Hi @thibaudcolas! I didn't understand the question, sorry. I just considered the `submit_line.html` file, which includes the 3 `input` `type=submit` "buttons" or there can be others?

Reply:
No worries! I think I’m probably overthinking this to be honest. I’m surprised we’re using `<input type="submit">` for this rather than `<button type="submit"></button>`, but  we’ll worry about this when we get there.